builtin/aws/ecs/platform.go

```diff
@@ -1524,6 +1529,9 @@ type Config struct {
 	// Subnets to place the service into. Defaults to the subnets in the default VPC.
 	Subnets []string `hcl:"subnets,optional"`
 
+	// Security Group ID to define existing security groups for ecs.
+	SecurityGroups []*string `hcl:"security_groups,optional"`
```

Comment:
At the risk of being pedantic, should this be `SecurityGroupIDs`?

Reply:
Agreed!